package.json

```diff
@@ -70,6 +69,7 @@
     "nodemon": "^1.10.2",
     "postcss-loader": "^0.13.0",
     "pre-commit": "^1.1.3",
+    "purecss": "^0.6.0",
     "react": "^15.3.1",
     "react-codemirror": "^0.2.6",
     "react-dom": "^15.3.1",
```

Comment:
quick one, are we really using all these dependencies? shall we run an `npm-check` just in case?